### PR TITLE
Additional checks before going into fallback

### PR DESCRIPTION
**Pull request by suhail-sullad**
For #16 we need to check whether there is a null or empty response before actually going into runBlocker. Otherwise redirect to runFallbackBlocker

@keraf  please do review and comment if any.

### CODE REVIEW

**Comment by keraf** [comment]
Thanks for the PR, looks good. I will test it tonight and merge it if everything is fine.

Could you also use the dev branch for this and any future PRs please? Thanks!

File: src/js/background.js [hunk 1]
```diff
@@ -135,8 +135,13 @@ if (!config.toggle) {
 const blacklist = 'https://raw.githubusercontent.com/keraf/NoCoin/master/src/blacklist.txt';
 fetch(blacklist)
     .then(resp => {
-        if (resp.status === 200) {
-            resp.text().then(text => runBlocker(text));
+        if (resp.status === 200 ) {
+            resp.text().then((text) => {
+				if(text === "" || text == null)
```
Comment (keraf) [inline src/js/background.js]: Just a few notes regarding the code:
- The indentation looks incorrect. Make sure you are using 4 spaces.
- Text should always be a string and therefore null checking is not necessary.
- For consistency, use single quotes across the project.
- For consistency again, use curly brackets for all conditions.

I will add linting rules in the future.

**Comment by suhail-sullad** [comment]
updated as per your comments.

**Comment by keraf** [comment]
Looks good, thanks!

**Pull request merged**